src/modules/launcher/Wox.Test/FuzzyMatcherTest.cs

```diff
@@ -122,21 +122,37 @@ public void WhenGivenStringsAndAppliedPrecisionFilteringThenShouldReturnGreaterT
             }
         }
 
-        [TestCase(Chrome, Chrome, 137)]
-        [TestCase(Chrome, LastIsChrome, 83)]
-        [TestCase(Chrome, HelpCureHopeRaiseOnMindEntityChrome, 21)]
-        [TestCase(Chrome, UninstallOrChangeProgramsOnYourComputer, 15)]
-        [TestCase(Chrome, CandyCrushSagaFromKing, 0)]
-        [TestCase("sql", MicrosoftSqlServerManagementStudio, 56)]
-        [TestCase("sql  manag", MicrosoftSqlServerManagementStudio, 79)]//double spacing intended
-        public void WhenGivenQueryStringThenShouldReturnCurrentScoring(string queryString, string compareString, int expectedScore)
+        [Test]
+        public void WhenMultipleResults_ExactMatchingResult_ShouldHaveGreatestScore()
         {
-            // When, Given
+            // Arrange
+            string queryString = "vim";
```

Comment:
This could be data driven for other test cases.    

eg. 
```

[TestCase("vim",  "vim", "ignoreDescription", "ignore.exe", "vim diff", "ignoreDescription", "ignore.exe")]
public void WhenMultipleResults_NameWithExactMatch_ShouldHaveGreatestScore(string query, string firstName, string firstDescription, string firstExecutableName, string secondName, string secondDescription, string secondExecutableName);